braph2genesis/measures/_MultilayerCommunity.gen.m

```diff
@@ -0,0 +1,816 @@
+%% ¡header!
+MultilayerCommunity < Measure (m, multilayer community structure) is the graph multilayer community structure.
+
+%%% ¡description!
+The multilayer community structure of a multilayer graph is a subdivision of 
+the network into non-overlapping groups of nodes which maximizes the number 
+of within group edges, and minimizes the number of between group edges.
+
+%% ¡layout!
+    
+%%% ¡prop!
+%%%% ¡id!
+MultilayerCommunity.ID
+%%%% ¡title!
+Measure ID
+
+%%% ¡prop!
+%%%% ¡id!
+MultilayerCommunity.LABEL
+%%%% ¡title!
+Measure NAME
+
+%%% ¡prop!
+%%%% ¡id!
+MultilayerCommunity.G
+%%%% ¡title!
+Graph
+
+%%% ¡prop!
+%%%% ¡id!
+MultilayerCommunity.M
+%%%% ¡title!
+MultilayerCommunity
+
+%%% ¡prop!
+%%%% ¡id!
+MultilayerCommunity.PFM
+%%%% ¡title!
+Measure Plot
+
+%%% ¡prop!
+%%%% ¡id!
+MultilayerCommunity.NOTES
+%%%% ¡title!
+Measure NOTES
+
+%%% ¡prop!
+%%%% ¡id!
+MultilayerCommunity.COMPATIBLE_GRAPHS
+%%%% ¡title!
+Compatible Graphs
+
+%% ¡props_update!
+
+%%% ¡prop!
+NAME (constant, string) is the name of the multilayer community structure.
+%%%% ¡default!
+'MultilayerCommunity'
+
+%%% ¡prop!
+DESCRIPTION (constant, string) is the description of the multilayer community structure.
+%%%% ¡default!
+'The multilayer community structure of a multilayer graph is a subdivision of the network into non-overlapping groups of nodes which maximizes the number of within group edges, and minimizes the number of between group edges.'
+
+%%% ¡prop!
+TEMPLATE (parameter, item) is the template of the multilayer community structure.
+%%%% ¡settings!
+'MultilayerCommunity'
+
+%%% ¡prop!
+ID (data, string) is a few-letter code of the multilayer community structure.
+%%%% ¡default!
+'MultilayerCommunity ID'
+
+%%% ¡prop!
+LABEL (metadata, string) is an extended label of the multilayer community structure.
+%%%% ¡default!
+'MultilayerCommunity label'
+
+%%% ¡prop!
+NOTES (metadata, string) are some specific notes about the multilayer community structure.
+%%%% ¡default!
+'MultilayerCommunity notes'
+
+%%% ¡prop!
+SHAPE (constant, scalar) is the measure shape __Measure.NODAL__.
+%%%% ¡default!
+Measure.NODAL
+
+%%% ¡prop!
+SCOPE (constant, scalar) is the measure scope __Measure.UNILAYER__.
+%%%% ¡default!
+Measure.UNILAYER
+
+%%% ¡prop!
+PARAMETRICITY (constant, scalar) is the parametricity of the measure __Measure.NONPARAMETRIC__.
+%%%% ¡default!
+Measure.NONPARAMETRIC
+
+%%% ¡prop!
+COMPATIBLE_GRAPHS (constant, classlist) is the list of compatible graphs.
+%%%% ¡default!
+{'MultiplexBU' 'MultiplexBUT'} ;%TBE % % % add any missing tests
```

Comment:
Add all the other graphs that are compatible (all multilayer, multiplex, ordml and ordmx)